doc/internal/release_process.rst

```diff
@@ -5,37 +5,57 @@ This process is only meant for the project administrators, not users and develop
 
-3. In that release branch, update the version numbers in ``version.py``, update the conda-recipe, and update the README, if necessary. You should also run `make linkcheck` on the documentation to fix and update any broken/redirected links.
+3. In the release branch:
 
-4. Upload source and wheel packages to PyPI using ``python setup.py sdist upload`` and ``python setup.py bdist_wheel upload``
+   a. update the version numbers in ``version.py``.
 
-5. Build the new generic conda package locally on your mac using the following command::
+   b. update the conda recipe.
 
-    conda build -c conda-forge rsmtool
+   c. update the documentation with any new features or details about changes.
 
-6. Upload the built package to anaconda.org using ``anaconda upload --user ets <package tarball>``.
+   d. run ``make linkcheck`` on the documentation and fix any redirected/broken links.
 
-7. Create pull requests on the `rsmtool-conda-tester <https://github.com/EducationalTestingService/rsmtool-conda-tester/>`_ and `rsmtool-pip-tester <https://github.com/EducationalTestingService/rsmtool-pip-tester/>`_ repositories to test the conda and PyPI packages on Linux and Windows.
+   e. update the README and this release documentation, if necessary.
 
-8. Draft a release on GitHub while the Linux and Windows builds are running.
+4. Build the PyPI source distribution using ``python setup.py sdist build``.
 
-9. Once both builds have passed, make a pull request with the release branch to be merged into ``master`` and request code review.
+5. Upload the source distribution to TestPyPI  using ``twine upload --repository testpypi dist/*``. You will need to have the ``twine`` package installed and set up your ``$HOME/.pypirc`` correctly. See details `here <https://packaging.python.org/guides/using-testpypi/>`__.
 
-10. Once the build for the PR passes and the reviewers approve, merge the release branch into ``master``.
+6. Install the TestPyPI package as follows::
 
-11. Make sure that the ReadTheDocs build for ``master`` passes.
+    pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple rsmtool
 
-12. Tag the latest commit in ``master`` with the appropriate release tag and publish the release on GitHub.
+7. Then run some tests from a RSMTool working copy. If the TestPyPI package works, then move on to the next step. If it doesn't, figure out why and rebuild and re-upload the package.
 
-13. Make another PR to merge ``master`` branch into ``stable`` so that the ``stable`` ReadTheDocs build always points to the latest release.
+8. Build the new generic conda package by running the following command in the ``conda-recipe`` directory::
 
-14. Update the CI plan for RSMExtra (only needed for ETS users) to use this newly built RSMTool conda package. Do any other requisite changes for RSMExtra. Once everything is done, do a release of RSMExtra.
+    conda build -c conda-forge -c ets .
```

Comment:
When I try to build the package, I get an error:

```
$ conda-recipe git:(release/7.1.0) ✗ pwd; conda build -c conda-forge -c ets .
/Users/mmulholland/Documents/rsmtool_dev/rsmtool_repo/conda-recipe
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.11
Copying /Users/mmulholland/Documents/rsmtool_dev/rsmtool to /Users/mmulholland/usr/local/share/conda/conda-bld/rsmtool_1582580520032/work/
Traceback (most recent call last):
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/utils.py", line 515, in _copy_with_shell_fallback
    stderr=subprocess.PIPE, stdout=subprocess.PIPE)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/subprocess.py", line 347, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'cp -a /Users/mmulholland/Documents/rsmtool_dev/rsmtool /Users/mmulholland/usr/local/share/conda/conda-bld/rsmtool_1582580520032/work/rsmtool' returned non-zero exit status 1.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/mmulholland/usr/local/share/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 469, in main
    execute(sys.argv[1:])
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 460, in execute
    verify=args.verify, variants=args.variants)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/build.py", line 2322, in build_tree
    bypass_env_check=True)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/render.py", line 802, in render_recipe
    try_download(m, no_download_source=no_download_source)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/render.py", line 628, in try_download
    source.provide(metadata)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/source.py", line 691, in provide
    locking=metadata.config.locking, clobber=True)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/utils.py", line 585, in copy_into
    _copy_with_shell_fallback(src, dst_fn)
  File "/Users/mmulholland/usr/local/share/conda/lib/python3.7/site-packages/conda_build/utils.py", line 518, in _copy_with_shell_fallback
    raise OSError("Failed to copy {} to {}.  Error was: {}".format(src, dst, e))
OSError: Failed to copy /Users/mmulholland/Documents/rsmtool_dev/rsmtool to /Users/mmulholland/usr/local/share/conda/conda-bld/rsmtool_1582580520032/work/rsmtool.  Error was: Command 'cp -a /Users/mmulholland/Documents/rsmtool_dev/rsmtool /Users/mmulholland/usr/local/share/conda/conda-bld/rsmtool_1582580520032/work/rsmtool' returned non-zero exit status 1.
```

However, when I remove one level from the `meta.yaml` `path` section (e.g. `path: ../../rsmtool` instead of `path: ../../../rsmtool`), the same command does not have that issue. I wonder where the command is being run from in its current state.